dealership/contracts.py

```diff
@@ -1,12 +1,50 @@
 class Contract(object):
-    pass
+    
+    stored_total_value = 0
 
 
 class BuyContract(Contract):
     def __init__(self, vehicle, customer, monthly_payments):
-        pass
+        self.vehicle = vehicle
+        self.customer = customer
+        self.monthly_payments = monthly_payments
+        
+        if self.customer.is_employee():
+            self.discount = .90
+        else:
+            self.discount = 1
+        
+    
+    def total_value(self): # total value of contract paid by customer
+        
+        total = (self.vehicle.sale_price() + (self.vehicle.interest_rate * \
+        self.monthly_payments * self.vehicle.sale_price() / 100)) * self.discount
```

Comment:
can we indent this for readability?

Reply:
I'll try that to see if it works...I bet it will.